### PR TITLE
add github username

### PR DESCRIPTION
**Pull request by YuriyLarin**

### CODE REVIEW

**Review by Virass — CHANGES_REQUESTED** [review]
Task not completed. Push all files to this PR, run test for task and add demo links to description

File: src/index.html [hunk 1]
```diff
@@ -11,7 +11,165 @@
...
+          <!-- <input
+            type="text"
+            name="fCar"
+            list="fCar"
+          >
+          <datalist id="fCar">
+            <option value="BMW">BMW</option>
+            <option value="Audi">Audi</option>
+            <option value="Lada">Lada</option>
+          </datalist><br> -->
```
Comment (loralevitska) [inline src/index.html:125]: remove comments

**Review by loralevitska — CHANGES_REQUESTED** [review]
Please deploy your solution, and before review check demo_link

File: src/index.html (hunk 1 above)
Comment (YuriyLarin) [reply src/index.html:125]: Done

Comment (YuriyLarin) [reply src/index.html]: done

File: src/style.css [hunk 2]
```diff
@@ -1 +1,18 @@
...
+}
+
+fieldset {
+  margin-bottom: 20px;
+}
+
+#yes,
+#no {
+  margin-bottom: 14px;
+}
```
Comment (SanyaBratashchuk) [inline src/style.css:18]: use classes for styling purposes

File: src/index.html [hunk 3]
```diff
@@ -11,7 +11,156 @@
...
+
+      <fieldset>
+        <legend>Registration</legend>
+        <label for="email" >E-mail:</label>
+        <input
+          type="email"
+          name="email"
+          id="email"
+          placeholder="email@example.com"
+        ><br>
```
Comment (SanyaBratashchuk) [inline src/index.html:70]: get rid of this `br`

Comment (SanyaBratashchuk) [reply src/index.html:70]: You just can wrap each input with label and apply `display: block` for it

**Review by SanyaBratashchuk — CHANGES_REQUESTED** [review]

**Pull request closed without merging**